tests/testthat/test-get_contact_matrix.R

```diff
@@ -0,0 +1,12 @@
+#testing for the get_contact_matrix.R function
+test_that("get_contact_matrix produces expected 21x21 square matrix output", {
+
+  #load("./data/contact_home.rda")
+  #run the function with Great Britain and contact_home
+  home_matrix <- get_contact_matrix("United Kingdom of Great Britain"
+                                    ,contact_home, 21)
```

Comment:
```suggestion
  home_matrix <- get_contact_matrix("United Kingdom of Great Britain",
                                    contact_home, 21)
```
I think this will remove the lint warning.